tests/js/web/FixedScrollBarComponentTest.js

```diff
@@ -0,0 +1,158 @@
+import $ from 'jquery';
+import StickyScrollBarComponent from '../../../js/StickyScrollBarComponent'
+
+describe('StickyScrollBarComponent', () => {
+	let $window;
+	let $stylesheet;
+	let $body;
+	let $container;
+	let $content;
+	let stickyScrollBarComponent;
+	let stickyScrollBarComponentWithoutWindow;
+	let stickyScrollBarComponentWithoutHtml;
+	let stickyScrollBarComponentWithoutContainer;
+
+	beforeEach(() => {
+		$window = $('<div></div>');
+        $window.height(500);
+		$stylesheet = $(`
+<style type="text/css">
+    #fakeBody {
+        width: 1000px;
+        height: 800px;
+    }
+
+    #fakeBody .container {
+        width: 500px;
+        height: 600px;
+        overflow: auto;
+    }
+
+    #fakeBody .content {
+        width: 1000px;
+        height: 200px;
+    }
+
+    #fakeBody .sticky-scrollbar {
+    	bottom: 0;
+    	height: 30px;
+    	overflow-x: auto;
+    	overflow-y: hidden;
+    	position: fixed;
+    	z-index: 9999;
+	}
+
+	#fakeBody .sticky-scrollbar__inner {
+    	border: 1px solid #fff;
+    	opacity: .01;
+	}
+
+	#fakeBody .footer {
+		bottom: 37px;
+		height: 37px;
+		position: fixed;
+	}
+</style>
+`).appendTo('head');
+
+		$body = $('<div id="fakeBody"><div class="container"><div class="content">some content</div></div></div>').appendTo('body');
+		$container = $body.find('.container');
+		$content = $body.find('.content');
+
+		stickyScrollBarComponent = new StickyScrollBarComponent($window, $body);
+	});
+
+	afterEach(() => {
+        $stylesheet.remove();
+        $body.remove();
+    });
+
+    describe('init()', () => {
+
+		it('should throw an error if $window isn\'t passed to the component', () => {
```

Comment:
As a sneaky sneaky thing, you could use the `’` character here instead of `\'`

⌥ + ⇧ + ]